server/server.js

```diff
@@ -161,11 +162,36 @@ module.exports = function(port, db, githubAuthoriser) {
                     contents: message,
                     timestamp: timestamp
                 }).then(function(result) {
+                    // Update conversation timestamp
                     conversations.updateOne({
                         _id: conversationID
                     }, {
                         $set: {lastTimestamp: timestamp}
                     });
+                    // Upsert a "new_messages" notification for each participant
```

Comment:
Ah, I found it after a quick search. You can undo your commit that clarifies it if you want. It was clear that what it was doing, I just wasn't sure if the word was commonly used.